Dockerfile

```diff
@@ -17,9 +17,10 @@ FROM $base_image
 ENV GOVUK_APP_NAME=asset-manager
 
 # TODO: move ClamAV into a completely separate service.
-RUN install_packages clamav clamav-daemon clamdscan shared-mime-info && \
-    rm -fr /etc/clamav/* && \
-    mkdir -p /var/run/clamav && \
+RUN install_packages wget shared-mime-info && \
+    wget https://www.clamav.net/downloads/production/clamav-1.2.1.linux.x86_64.deb && \
```

Comment:
I'd recommend making the filename a variable (`ARG`) so that you can update in one place.